src/SCSSolverInterface.jl

```diff
@@ -281,8 +285,8 @@ function orderconesforscs(A_in, b_in, c_cones, v_cones)
             b = [b; b_in[idxs,:]]
             # n must be a square integer
             n = length(idxs)
-            isinteger(sqrt(n)) || error("number of SDP variables must be square")
-            sqrt_n = convert(Int, sqrt(n));
+            isinteger(invertsdconesize(n)) || error("number of SDP variables must be n*(n+1)/2")
```

Comment:
Ok thanks. Would that be an issue with the sqrt() call that was there before too?